mcs/class/System.Windows.Forms/System.Windows.Forms/GridEntry.cs

```diff
@@ -792,7 +784,7 @@ private bool IsPropertyMergeable (PropertyDescriptor property)
 		{
 			if (property == null)
 				return false;
-				
+
```

Comment:
Reduce line damage. Revert that change.

Reply:
@luhenry, I guess this change does not damage line number. I have removed meaningless tab symbols without line number changes. Totally empty lines (without any kind of spaces) is a normal condition for the current file. IMHO, of course :)